scripts/sanitycheck

```diff
@@ -3710,34 +3737,71 @@ def native_and_unit_first(a, b):
         return -1
     if b[0].startswith('native_posix'):
         return 1
-    if a[0].split("/",1)[0].endswith("_bsim"):
+    if a[0].split("/", 1)[0].endswith("_bsim"):
         return -1
-    if b[0].split("/",1)[0].endswith("_bsim"):
+    if b[0].split("/", 1)[0].endswith("_bsim"):
         return 1
 
     return (a > b) - (a < b)
 
 
-run_individual_tests = None
-options = None
+class HardwareMap:
+    manufacturer = [
```

Comment:
For the Microchip board necessary to add a support, because I found out that I can't create a hardware map for that board, it is not supported. That board uses runner `dediprog` for west flash, but even in current Sanitycheck script it is not added.
So necessary to add one more option `Microchip`